public/approval/v1.0/openapi.json

```diff
@@ -1050,27 +1076,27 @@
                         "$ref": "#/components/schemas/ActionIn"
                     },
                     {
-                        "type": "object",
-                        "required": [
-                            "id",
-                            "stage_id"
-                        ],
-                        "properties": {
-                            "id": {
-                                "type": "string"
-                            },
-                            "created_at": {
-                                "type": "string",
-                                "format": "date-time",
-                                "description": "Timestamp of creation"
-                            },
-                            "stage_id": {
-                                "type": "string",
-                                "description": "Associated stage id"
-                            }
-                        }
+                        "type": "object"
```

Comment:
@lindgrenj6 what's the reason you made these changes? reformat?

Reply:
Yes, when defining the `Out` objects initially it looks like we used `allOf` which includes fields, and then after that declared the extra fields. That _can_ go in the same block as `allOf` but that makes it look like the the object has no fields. So all I did was move the extra field declarations outside of the `allOf` part, so they show up the way the `Api::Doc` parser is expecting.

Reply:
👍